### PR TITLE
integrations/kubernetes: add clarifications to update instructions

### PR DESCRIPTION
**Pull request by roobre**
### Tell us why

This PR adds clarifications to the update section, as previously some customers tried to follow upgrade instructions meant for custom installations when they had used the automated installer.

We realized this section was a bit ambiguous, so this clarified that these instructions only apply to custom manifests, and pointed to upgrade instrucitons for the other two methods we support.

### Anything else you'd like to share?

Additionally, my linter was yelling at me with a lot of syntax errors due to some placeholders enclosed in `<brackets>`. I have escaped them (`<>`) as `&lt;&gt;`). If this is not the recommended way, please tell me and I'll drop the commit.

### CODE REVIEW

File: src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx [hunk 1]
```diff
@@ -91,21 +91,21 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/nr1-
   >
     To deploy the Kubernetes integration with [OpenShift](https://learn.openshift.com/?extIdCarryOver=true&sc_cid=701f2000001OH7iAAG):
 
-    1. Add the `<release_name>-newrelic-infrastructure` service account to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
+    1. Add the `<>{'<release_name>'}</>-newrelic-infrastructure` service account to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
 
        ```
        oc adm policy add-scc-to-user privileged \
-       system:serviceaccount:<mark><namespace></mark>:<release_name>-newrelic-infrastructure
+       system:serviceaccount:<mark>&lt;namespace&gt;</mark>:<>{'<release_name>'}</>-newrelic-infrastructure
```
Comment (roobre) [inline src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:98]: I am not entirely surre about this one. Is there any way to check if this is rendering correctly?

Comment (mmfred) [reply src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx:98]: Hi @roobre There is a way to check this. That's to build the project locally. I'll do that, and share a screenshot with you, but for future reference, this section of the contributor guide describes how to view your changes locally:
https://github.com/newrelic/docs-website/blob/develop/CONTRIBUTING.md#local-development

**Comment by mmfred** [comment]
@roobre here's what those escaped snippets look like - is this correct: [
escape_brackets
escape-snippet

](url)

**Comment by mmfred** [comment]
@bradleycamacho - here's the PR I mentioned.

**Comment by roobre** [comment]
@mmfred Thanks for checking this, and sorry for the mess. Seems like the final system and my linter are not very well aligned.

Feel free to rebase or drop the commit (https://github.com/newrelic/docs-website/pull/1085/commits/6297bf48c8b152944360989325cbb23ff5e27438) and leave it unchanged, or ping me if you prefer me to do it.

**Comment by bradleycamacho** [comment]
@roobre No need to apologize! I'll get to work on this and get it merged later today :)

**Comment by roobre** [comment]
Hey, is there anything pending on my side for this? Thanks!

**Pull request merged**